### PR TITLE
cpuid: update to 20201006.

### PR DESCRIPTION
**Pull request by arete**

### CODE REVIEW

**Comment by codingHahn** [comment]
Could you update the checksum as well? Just run `xgensum -i srcpkgs/cpuid/template`

File: srcpkgs/cpuid/template [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Gerardo Di Iorio  <arete74@gmail.com>"
 license="GPL-2.0-or-later"
 homepage="http://www.etallen.com/cpuid.html"
 distfiles="http://www.etallen.com/cpuid/cpuid-${version}.src.tar.gz"
-checksum=c1a9463f1f2175086120e9079e318bbd383dc1e574fafb2f724879f1d11061d4
+checksum=13c53f16408341d11ed489b352f218ec4dbb5bedb796a9d059e924770a9c56a6
 
 post_extract() {
 	sed -i 's|BUILDROOT=|BUILDROOT=${DESTDIR}|' Makefile
```
Comment (ericonr) [inline srcpkgs/cpuid/template:16]: Please switch to using `vsed`.

**Comment by ericonr** [comment]
Have you tested the package locally? The checksum error should have been caught.

You don't need to become a package maintainer if you just want to update the package. However, feel free to do so :)

**Comment by arete** [comment]
@ericonr hi have fix the template, compiled  work fine on my pc

**Pull request merged**